libvips/foreign/cgifsave.c

```diff
@@ -601,6 +602,16 @@ vips_foreign_save_cgif_write_frame( VipsForeignSaveCgif *cgif )
 		frame_config.numLocalPaletteEntries = n_colours;
 	}
 
+	/* Write an interlaced GIF, if requested.
+	*/
+	if( cgif->interlace ) {
+#ifdef HAVE_CGIF_FRAME_ATTR_INTERLACED
+		frame_config.attrFlags |= CGIF_FRAME_ATTR_INTERLACED;
+#else /*!HAVE_CGIF_FRAME_ATTR_INTERLACED*/
+		vips_error( class->nickname, "%s", _( "cgif >= v0.3.0 required for interlaced GIF write" ) );
+#endif /*HAVE_CGIF_FRAME_ATTR_INTERLACED*/
```

Comment:
This should probably be a warning, or if this is an error, `return -1;` after.

Reply:
> This should probably be a warning, or if this is an error, `return -1;` after.

Sounds good. I changed it to be just a warning.